docs/7.x/changelog.md

```diff
@@ -54,6 +54,21 @@ extend updates past End of Support through customer agreements if required.
 
 7.1 is currently pre-release.
 
+### 7.1.0-alpha.6 (April 14, 2021)
+
+#### Improvements
+* Add GCE Alias IP support for flannel ([#2439](https://github.com/gravitational/gravity/pull/2439), [planet#830](https://github.com/gravitational/planet/pull/830), [flannel#10](https://github.com/gravitational/flannel/pull/10))
```

Comment:
I think we put `.` at the end of each entry, although I think alpha.5 is also missing this.

Reply:
Added to this changelog in 6aa4ac2.  I'll catch the others in a later PR as I don't want to include unrelated changes from a diff tracking perspective.